lib/addons/keda/index.ts

```diff
@@ -0,0 +1,143 @@
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi";
+import { setPath, createNamespace } from "../../utils";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface KedaAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Version of the helm chart to deploy
+     */    
+    version?: string,
+    /**
+     * Name of the KEDA operator
+     */
+    kedaOperatorName?: string,
+    /**
+     * Specifies whether a service account should be created by by CDK with IRSA. If provided false, Service Account will be created by Keda without IRSA
+     */
+     enableIRSA?: boolean,
+     /**
+      * Specifies whether a service account should be created by Keda, by default its true, but in case IRSA is true it will be set to false automatically
+      */
+     createServiceAccount?: boolean,
+    /**
+     * The name of the service account to use. If not set and create is true, a name is generated.
+     */
+    kedaServiceAccountName?: string,
+    /**
+     * securityContext: fsGroup
+     * Check the workaround for SQS Scalar with IRSA https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+     */   
+    podSecurityContextFsGroup?: number,
+    /**
+     * securityContext:runAsUser
+     * Check the workaround for SQS Scalar with IRSA https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+     */   
+    securityContextRunAsGroup?: number,
+    /**
+     * securityContext:runAsGroup
```

Comment:
minor: I assume you meant runAsUser here.

Reply:
fixed